config/s3/s3_object_test.yaml

```diff
@@ -681,3 +681,13 @@ test_7657:
   bucket_name: "objworkflow7657"
   no_of_objects: 1010
   del_obj_cnt: 1000
+test_42776:
+  object_tag: "testkey=organizationCreateatagwhosekeyisupto128UnicodecharactersinlengtCreateatagwhosekeyisupto128Unicodecharacterslengthshouldbe128charorganizationCreateatagwhosekeyisupto128UnicodecharactersinlengtCreateatagwhosekeyisupto128Unicodecharacterslengthshouldbe128char"
+  tag_count: 1
+test_42777:
+  object_tag: "organizationCreateatagwhosekeyisupto128UnicodecharactersinlengtCreateatagwhosekeyisupto128Unicodecharacterslengthshouldbe128char=valuekey"
+  value: "valuekey"
```

Comment:
Maybe this line (689) can be removed if it is not used anywhere in the test

Reply:
Removed.